packages/twenty-front/src/modules/action-menu/actions/record-actions/single-record/constants/DefaultSingleRecordActionsConfigV2.ts

```diff
@@ -27,13 +29,25 @@ export const DEFAULT_SINGLE_RECORD_ACTIONS_CONFIG_V2: Record<
     actionHook: SingleRecordActionHook;
   }
 > = {
+  exportNoteToPdf: {
+    type: ActionMenuEntryType.Standard,
+    scope: ActionMenuEntryScope.RecordSelection,
+    key: SingleRecordActionKeys.EXPORT_NOTE_TO_PDF,
+    label: 'Export note',
```

Comment:
Rename to `Export to PDF`? It's also displayed on tasks

Reply:
![image](https://github.com/user-attachments/assets/ef553676-a09e-457b-a182-bdcdeb8a7818)

I named it 'note' because the the RightDrawer shows the editor under the Note tab within Task.

Reply:
Resolved in recent commit, but leaving this here for review.

Reply:
Ah I see thanks!